modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPDefinitionLocalServiceImpl.java

```diff
@@ -2012,7 +2013,12 @@ public boolean isVersionable(CPDefinition cpDefinition) {
 
 	@Override
 	public boolean isVersionable(long cpDefinitionId) {
-		if (!_isVersioningEnabled()) {
+		CPDefinition cpDefinition = cpDefinitionLocalService.fetchCPDefinition(
```

Comment:
this method could call the one with the CPDefinition cpDefinition as parameter (or the object) instead of replicate all the logic